test/spec/services/printutils.spec.js

```diff
@@ -2,11 +2,11 @@ goog.require('ngeo.print.Utils');
 
 describe('ngeo.print.Utils', () => {
 
-  let ngeoPrintUtils;
+  let ngeoPrintUtilsService;
 
   beforeEach(() => {
-    angular.mock.inject((_ngeoPrintUtils_) => {
-      ngeoPrintUtils = _ngeoPrintUtils_;
+    angular.mock.inject((ngeoPrintUtils) => {
+      ngeoPrintUtilsService = ngeoPrintUtils;
```

Comment:
Is there a reason not to use anymore the underscore hack?

Reply:
This is not for removing the _ but to not overriding the import in es6 mode...